### ISSUE TITLE
Getting sink pointer for an asynchronous logger

### ISSUE THREAD

**Issue by Heshan94** [gabime/spdlog#2068]
How can I get a pointer for the sink after making it as an asynchronous logger. My expectation is to push multiple sinks into spdlog::logger::sinks(), But after I create an asynchronous logger like below, is there way to push it to the logger? 
If it isn't possible how can push asynchronous and non-asynchronous sinks to logger without making all sinks asynchronous?

`auto async_file = spdlog::create_async<spdlog::sinks::basic_file_sink_mt>("async_file_logger", "logs/async_log.txt")`

Basically what I'm trying to do is similar to this.
`spdlog::get("async_file_logger")->sinks().push_back(async_file);`

**Comment by tt4g** [comment]
It is recommended to use `spdlog::sinks::dist_sink_mt`.
